assets/src/stories-editor/components/with-active-page-state.js

```diff
@@ -32,6 +35,7 @@ const withActivePageState = ( BlockListBlock ) => {
 				...props.className,
 				'amp-page-active': isTopLevelBlock && isActivePage,
 				'amp-page-inactive': isTopLevelBlock && ! isActivePage,
+				'amp-page-is-animating': isPlayingAnimation,
```

Comment:
If this is only for active page then what happens if the user moves to another page while the animation is playing?

Reply:
Looks like this happens:
<img width="882" alt="test" src="https://user-images.githubusercontent.com/3294597/64169926-25987f00-ce4f-11e9-820d-0dd4fa050cf0.png">

Reply:
That looks fun! Let's leave it like this 😄 

Seems like I need to move all of the animation playback stuff to the data store, so that we know exactly which animations are currently playing where.

Then, when doing an action like changing the current page we can easily cancel them via the data store.